tools/releaser/lib/version_updater.dart

```diff
@@ -157,3 +161,33 @@ Future<bool> _updateReadmeVersions(CommandArguments args, Logger logger) async {
 
   return true;
 }
+
+Future<bool> _updateNativeSDKVersions(CommandArguments args, Logger logger) async {
+  final nativeSDKVersionsFile = File(path.join(args.packageRoot, 'NATIVE_SDK_VERSIONS.md'));
+  final newVersionEntry ='| ${args.version} | ${args.iOSRelease} | ${args.androidRelease} |';
+  final header = '| Flutter | iOS SDK | Android SDK |';
+  final separator = '|---------|---------|-------------|';
+
+  if (!nativeSDKVersionsFile.existsSync()) {
+    logger.warning('⚠️ NATIVE_SDK_VERSIONS.md does not exist, creating it now.');
+    await nativeSDKVersionsFile.writeAsString('''$header\n$separator\n$newVersionEntry''');
+    return true;
+  }
+
+  final content = await nativeSDKVersionsFile.readAsString();
+
+  if (content.contains(newVersionEntry)) {
```

Comment:
I don't think this is right. It will search for any version matching, not just the Flutter versions. It would fail when we go to ship Flutter SDK 2.11.0 for example because iOS SDK 2.11.0 is in the table.

Reply:
I agree, it is too specific and it only looks for a line containing all three versions together (as it uses  `newVersionEntry = '| ${args.version} | ${args.iOSRelease} | ${args.androidRelease} |';`). 

I have modified it so instead it looks for the Flutter SDK version across the whole file being present on the first column of the .md file.

changes at [0fa8f1c](https://github.com/DataDog/dd-sdk-flutter/pull/737/commits/0fa8f1c2dcc0659dcc1bd0c4a54d31d240d5c598)